css/Style.css

```diff
@@ -1 +1,147 @@
-li { color: blue; }
+html { box-sizing: border-box; }
+
+#header-background {
+  width: 375px;
+  height: 718px;
+
+  /* background-color: goldenrod; */
+  left: 0;
+  top: 0;
+}
+
+#flex-nav {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 375px;
+  height: 48px;
+
+  /* background-color: goldenrod; */
+}
+
+#flex-nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+
+  /* width: 375px;
+    height: 48px;
+    position: absolute;
+    margin-left: -30px; */
+}
+
+/* #burguer-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 24px;
+    left: 311px;
+    top: 4px;
+    border-radius: 8px;
+    /* background-color: aqua;
+} */
```

Comment:
[OPTIONAL] Please do not leave unused codes as comments in your codebase since it will make it bulky and slow.